content/docs/introduction/faq.md

```diff
@@ -11,7 +11,7 @@ sort_rank: 5
 Prometheus is an open-source systems monitoring and alerting toolkit
 with an active ecosystem. See the [overview](/docs/introduction/overview/).
 
-### How does Prometheus compare against \[other monitoring system\]?
+### How does Prometheus compare against \[other monitoring systems\]?
```

Comment:
This wasn't actually a typo, but a placeholder (thus the square brackets). But I guess the way it's now is ok too.